pkg/sanity/controller.go

```diff
@@ -854,6 +855,45 @@ var _ = DescribeSanity("Controller Service", func(sc *SanityContext) {
 			cl.UnregisterVolume(name)
 		})
 
+		It("should fail when publishing volumes more than the node max attach limit", func() {
```

Comment:
"should fail when publishing more volumes than allowed by the node max attach limit"